torch/_higher_order_ops/invoke_subgraph.py

```diff
@@ -771,3 +802,115 @@ def _unwrap_proxy(arg):
     return track_tensor_tree(
         example_out, out_proxy, constant=None, tracer=proxy_mode.tracer
     )
+
+
+# For testing only!
+invoke_subgraph_inductor_compile_captured_gms = None
```

Comment:
If this is for testing, I suggest renaming the variable name to reflect that.

Reply:
added _testing in name